manifests/extensions/validating-webhook/rbac/argo-events-webhook-cluster-role.yaml

```diff
@@ -27,9 +27,11 @@ rules:
       - apps
     resources:
       - deployments
+      - deployments/finalizers
```

Comment:
1. I think this would be better to go with a notice in a doc for OpenShift installation.
2. From the document, it only requires update privilege on `xxx/finalizers`, not the  update privilege on the whole `xxx`.

Reply:
1. Alright, I'll add it to the docs.
2. You're right